sway/commands/force_display_urgency_hint.c

```diff
@@ -11,10 +11,12 @@ struct cmd_results *cmd_force_display_urgency_hint(int argc, char **argv) {
 	char *err;
```

Comment:
In fact, I'd prefer to name this variable `end` rather than `err`. `strtol` will set it to the first non-digit character.

Reply:
yes, I agree!